src/main/java/org/sbml/spatial/segmentation/TrainUnetModel.java

```diff
@@ -111,14 +114,14 @@ public static void main(String[] args) {
 
       int numEpochs = 1;
 
-      ComputationGraph model  = UNet.builder().updater(new Adam(1e-4)).build().init();
-      //	            ComputationGraph model  = UNet.builder().updater(new Adam(new MapSchedule(ScheduleType.ITERATION, learningScheduleMap))).build().init();
+      //ComputationGraph model  = UNet.builder().updater(new Adam(1e-4)).build().init();
+      //ComputationGraph model  = UNet.builder().updater(new Adam(new MapSchedule(ScheduleType.ITERATION, learningScheduleMap))).build().init();
       
       
       //To change the number of input channels from 3 to 1
-        //ZooModel unet = UNet.builder().build();
-	    //unet.setInputShape(new int[][]{{1, 128, 128}});
-	    //ComputationGraph model = (ComputationGraph) unet.init();
+        ZooModel unet = UNet.builder().build();
+	    unet.setInputShape(new int[][]{{1, 128, 128}});
+	    ComputationGraph model = (ComputationGraph) unet.init();
```

Comment:
Why did we change this part of the code?
Is it any relevant to the issue?

Reply:
There was an error...I had added this bit earlier for setting the number of input channels to 1. 
Not relevant to path....just to avoid an error that comes with using 1 input channel.

Reply:
Did this error occur due to the changes you made on this branch that are relevant to the issue? If so, it makes sense to incorporate this change. If not, this should be done in a separate branch.

Reply:
So, should I undo this change and make a separate pull request? Because it is not relevant to this particular issue (regarding path).

Reply:
Yes.

Reply:
Ok...will do so.